cloudstate/cloudstate.go

```diff
@@ -46,9 +46,13 @@ type CloudState struct {
 
 // New returns a new CloudState instance.
 func New(options Options) (*CloudState, error) {
```

Comment:
minor nitpick: should we call it `Config` instead of `Options`?

Reply:
yes, makes sense.

- [ ] call it Config instead of Options.